packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -62,9 +69,19 @@ export const makeVaultManager = (
   reallocateReward,
   timerService,
   liquidationStrategy,
+  startTimeStamp,
 ) => {
   const { brand: runBrand } = runMint.getIssuerRecord();
 
+  const { updater, notifier } = makeNotifierKit(
+    harden({
+      compoundedInterest: makeRatio(1n, runBrand, 1n, runBrand),
+      latestInterestUpdate: 0n,
```

Comment:
Is the identity for interest `1n` instead of `0n`?

Reply:
`latestInterestUpdate` is a timestamp, 0n here because there is no previous update. I'll add a comment.

Reply:
Yes! d2560baf790b76891a039d132ece224f52e88824